### DISCUSSION TITLE
Training for custom keypoints

### DISCUSSION THREAD

**Discussion by priyamw** [open-mmlab/mmpose#3109] [Q&A]
I am training a custom dataset for 22 keypoints, but my dataset is not generalized. If there is way to use coco 17 keypoints  weights to fine my custom model which will output 22 keypoints.

**Comment by Nik-su** [comment]
Hi @priyamw did you get any idea on training with the custom data with mmpose model. I am facing many issues while training. Can you please tell me how you solved this problem.

**Comment by priyamw** [comment]
yes, i have successfully fine-tuned the model (rtmpose)  for my dataset. https://mmpose.readthedocs.io/en/0.x/tutorials/1_finetune.html
this helped me if you have any specific query i will be happy to help.
